t/01-compile-check-all.t

```diff
@@ -34,7 +34,7 @@ BEGIN {
 }
 
 my $test = Test::Compile->new();
-$test->verbose(0);
+$test->verbose(1);
```

Comment:
unrelated

Reply:
I can split it into another commit. But I'd like to keep it because it is actually good to have this output.